app/controllers/stories_controller.rb

```diff
@@ -414,4 +414,10 @@ def user_same_as
       @user.profile.website_url,
     ].compact_blank
   end
+
+  def fetch_sort_order
+    return params[:comments_sort] if Comment::VALID_SORT_OPTIONS.include? params[:comments_sort]
```

Comment:
refactor, non-blocking: the logic here about which sort options are valid, and the "top" default sort option if not provided, might be moved to a method in the Comment model, passing the `:comment_sort` param to the method and receiving a string back. That keeps the http request details (params provided) in the controller, but the rules about sorting behavior in the model (where you added the constant).

The code smell here for me is reaching into the value of the class's constant, outside of a controller tied to that model like CommentsController.

Something like this might make sense. 

```ruby
class Comment
  def self.sort_order(option)
     # code from fetch_sort_order here    
  end
end
```

I'll merge this as-is, if this logic needs to change in the future we can revisit moving the decision making out of the controller.